schema.graphql

```diff
@@ -152,6 +152,30 @@ type PoolInteraction @entity(immutable: true) {
 
   "The total staked amount after interaction"
   totalStaked: BigInt!
+
+  "The current pool rate"
+  rate: BigInt!
+}
+
+type UserInteraction @entity(immutable: true) {
+  "The tx hash"
+  id: Bytes!
```

Comment:
We should handle the case when a user make multiple interactions during the same transaction. Maybe use a counter as in the PoolInteraction